api/dao/hierarchy.py

```diff
@@ -325,6 +325,8 @@ def _update_hierarchy(container, container_type, metadata):
         session_obj = None
         if session.keys():
             session['modified'] = now
+            if session.get('timestamp'):
+                session['timestamp'] = dateutil.parser.parse(session['timestamp'])
```

Comment:
@ryansanford - I found the culprit! Thanks for narrowing the search down for me.